.github/workflows/lint.yml

```diff
@@ -0,0 +1,35 @@
+name: ⬣ Lint
+
+on:
+  workflow_call:
+
+jobs:
+  lint:
+    name: ⬣ Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
```

Comment:
```suggestion
        uses: styfle/cancel-workflow-action@0.10.0
```

Reply:
we can handle this update in a follow up PR :)